packages/component-submission/server/aggregates/Submission.test.js

```diff
@@ -192,33 +176,287 @@ describe('Submission', () => {
       )
```

Comment:
I think we should have a unit test under the `updateManuscript` describe to check the status validation here: https://github.com/elifesciences/elife-xpub/pull/2009/files#diff-5b518139d4766e3a75dc9c2cb1213c90R84